cognite/client/data_classes/filters.py

```diff
@@ -30,7 +31,7 @@ class ParameterValue:
 
 
 FilterValue: TypeAlias = RawValue | PropertyReferenceValue | ParameterValue
-FilterValueList: TypeAlias = Sequence[RawValue] | PropertyReferenceValue | ParameterValue
+FilterValueList: TypeAlias = Sequence[RawValue | InstanceId] | PropertyReferenceValue | ParameterValue
```

Comment:
I believe we should add it to `RawValue` instead, then `FilterValue` and `FilterValueList` will include it automatically.
```py
RawValue: TypeAlias = str | float | bool | Sequence | Mapping[str, Any] | Label
```